README.md

```diff
@@ -715,6 +715,7 @@ API | Description | Auth | HTTPS | CORS |
 | [Pulsedive](https://pulsedive.com/api/) | Scan, search and collect threat intelligence data in real-time | `apiKey` | Yes | Unknown |
 | [SecurityTrails](https://securitytrails.com/corp/apidocs) | Domain and IP related information such as current and historical WHOIS and DNS records | `apiKey` | Yes | Unknown |
 | [Shodan](https://developer.shodan.io/) | Search engine for Internet connected devices | `apiKey` | Yes | Unknown |
+| [Spyse](https://spyse.com/api/) | Search engine for internet assets: Ready-made data on IPs, Domains, Vulnerabilities, etc | `apiKey` | Yes | Unknown |
```

Comment:
The link of `https://spyse.com/api/` is redirected to `https://spyse-dev.readme.io/reference/quick-start`. 
Can you please update the link?